src/briefcase/platforms/windows/__init__.py

```diff
@@ -354,10 +374,23 @@ def package_app(
         except subprocess.CalledProcessError as e:
             raise BriefcaseCommandError(f"Unable to link app {app.app_name}.") from e
 
-        if sign_app:
-            self.logger.info("Signing MSI...", prefix=app.app_name)
-            self.sign_file(
-                app=app,
-                filepath=self.distribution_path(app),
-                **sign_options,
-            )
+    def _package_zip(self, app):
+        """Package the app as simple zip file."""
+
+        self.logger.info("Building zip file...", prefix=app.app_name)
+        with self.input.wait_bar("Packing..."):
+            source = self.bundle_path(app) / self.packaging_root  # /src
+            try:
+                os.mkdir(self.dist_path)  # /dist
+            except FileExistsError:
```

Comment:
OK, this `except` is missing coverage.

@freakboy3742 @rmartin16: I saw that testing creates several temporary folders, for each test one (?). So when I want to test a condition that a folder was already created (like here), I need to test this within the same `test_...()` function, otherwise I end up in a fresh test folder, is this correct?

Reply:
> I saw that testing creates several temporary folders, for each test one (?).

Yes; Pytest provides the `tmp_path` fixture for each test such that it's a unique empty directory that's ultimately cleaned up by Pytest.

> So when I want to test a condition that a folder was already created (like here), I need to test this within the same `test_...()` function, otherwise I end up in a fresh test folder, is this correct?
> 
> Or I create the `dist` folder in the fixture from the beginning, so no need to run the `_package_zip()` two times.

I'd probably need to see a specific test example...but moreover, you shouldn't need to create the `dist_path` directory here; it is [already created](https://github.com/beeware/briefcase/blob/5c83de4a0967eef40b6145e42025fbe37d2ba540/src/briefcase/commands/package.py#L93) by the base `PackageCommand`.

Reply:
The mkdir call definitely isn't needed. It will be [created by the base package command](https://github.com/beeware/briefcase/blob/main/src/briefcase/commands/package.py#L93). That doesn't guarantee it exists for your *test* - so you may need to create the dist folder as part a fixture or other test setup.